<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEnter</class>
 <widget class="QDialog" name="DialogEnter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>184</width>
    <height>123</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter to the system</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>role</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_role">
      <item>
       <property name="text">
        <string>specialist</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>administrator</string>
       </property>
      </item>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_login"/>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogEnter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogEnter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
